cmd/test-bot.rb

```diff
@@ -71,6 +71,9 @@ def test_bot_args
              description: "Only run the local system setup check step."
       switch "--only-tap-syntax",
              description: "Only run the tap syntax check step."
+      switch "--stable",
+             depends_on:  "--only-tap-syntax",
+             description: "Only run the tap syntax checks needed on stable brew."
```

Comment:
Whoops, only thought more after merge: I think this is maybe a bit unclear for third-party taps as to what it's doing? What about making this `--skip-audit` instead?